hack/generate-manifest-windows.sh

```diff
@@ -95,49 +90,36 @@ fi
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-source $THIS_DIR/verify-kustomize.sh
+source $THIS_DIR/verify-helm.sh
 
-if [ -z "$KUSTOMIZE" ]; then
-    KUSTOMIZE="$(verify_kustomize)"
-elif ! $KUSTOMIZE version > /dev/null 2>&1; then
-    echoerr "$KUSTOMIZE does not appear to be a valid kustomize binary"
+if [ -z "$HELM" ]; then
+    HELM="$(verify_helm)"
+elif ! $HELM version > /dev/null 2>&1; then
+    echoerr "$HELM does not appear to be a valid helm binary"
     print_help
     exit 1
 fi
 
-KUSTOMIZATION_DIR=$THIS_DIR/../build/yamls/windows
-
-TMP_DIR=$(mktemp -d $KUSTOMIZATION_DIR/overlays.XXXXXXXX)
-
-pushd $TMP_DIR > /dev/null
-
-BASE=../../containerd
 if $INCLUDE_OVS; then
-    BASE=../../containerd-with-ovs
-fi
-
-mkdir $MODE && cd $MODE
-touch kustomization.yml
-# ../../patches/$MODE may be empty so we use find and not simply cp
-find ../../patches/$MODE -name \*.yml -exec cp {} . \;
-
-$KUSTOMIZE edit add base $BASE
-
-if [ "$MODE" == "dev" ]; then
-    $KUSTOMIZE edit set image antrea-windows=antrea/antrea-windows:latest
-    $KUSTOMIZE edit add patch --path imagePullPolicy.yml
+    HELM_VALUES+=("includeOVS=true")
```

Comment:
Is setting `includeOVS=true` redundant? It seems that it has been set to true in values.yaml. Please correct me if I'm wrong.

Reply:
Please see the discussion [here](https://github.com/antrea-io/antrea/pull/6360#discussion_r1626532376).

Reply:
> Please see the discussion [here](https://github.com/antrea-io/antrea/pull/6360#discussion_r1626532376).

Understood, thanks for the explanation.